### ISSUE TITLE
Add support for different date types (JD, MJD, HJD, BJD)

### ISSUE THREAD

**Issue by petro121** [ebachelet/pyLIMA#49]
Even though in pyLIMA there are no absolute definition for which date type to use, some researches might like to account for subtle differences in time for different date types (Julian Date, Modified Julian Date, Heliocentric Julian Date, Barycentric Julian Date). This need is produced by the fact that there are some subtle effects that are mildly impacted by this, such as the "Earth parallax".

**Comment by stelais** [comment]
I agree with Petro, that addition would be very helpful. Different telescopes and tools generate files with different "times" formats, and to run pyLIMA we end up having to create a second "uniformed" file, saving double information. I would also suggest adding "HJD-2450000" to the list of available formats when implemented. Same as Petro, this appeared to me when trying to use parallax.
